docs/reference/setup-tools/kubeadm/kubeadm-join.md

```diff
@@ -21,6 +21,13 @@ This action consists of the following steps:
    authenticity of that data. The root CA can also be discovered directly via a
    file or URL.
 
+1. If kubeadm is invoked with `--feature-gates=DynamicKubeletConfig` enabled,
+   first kubeadm retrives the kubelet init configuration from the master and writes is to
```

Comment:
> first kubeadm retrives the kubelet init configuration from the master and writes is to

it first retrieves the kubelet init configuration from the master and writes it to